converter/SEMSinput.py

```diff
@@ -86,6 +89,15 @@
     "8": {"name": "candidates", "fields": ["contest_id", "candidate_id", "label", "type", "sort_seq", "party_id", "label_on_ballot"]},
     "9": {"name": "sems_candidates", "fields": ["county_code", "contest_id", "candidate_id", "candidate_sems_id"]}
 }
+
+
+FULL_PARTY_NAMES = {
+    "democrat": "Democratic Party",
+    "nonpartisan": "Nonpartisan"
+}
+
+def full_party_name(short_party_name):
+    return FULL_PARTY_NAMES.get(short_party_name.lower(), short_party_name + " Party")
```

Comment:
Feels a little brittle, but fine for now.

Reply:
I don't think we'll be able to do much better automated, given the list of parties that comes out of SEMS looks quite stable from election to election, and if it changes we'd need to review manually anyways, I think.